test/benchee/formatters/console_test.exs

```diff
@@ -56,6 +65,73 @@ defmodule Benchee.Formatters.ConsoleTest do
   end
 
   describe ".format_scenarios" do
+    test "displays extended options" do
+      scenarios = [
+        %Scenario{
+          job_name: "First job",
+          run_time_statistics: %Statistics{
+            average: 200.0,
+            ips: 5_000.0,
+            std_dev_ratio: 0.1,
+            median: 195.5,
+            percentiles: %{99 => 300.1},
+            minimum: 111.1,
+            maximum: 333.3,
+            mode: 201.2,
+            sample_size: 50_000
+          }
+        }
+      ]
+
+      [_header1, _result1, descriptor1, descriptor2, result2] =
+        Console.format_scenarios(scenarios, @console_config_extended_params)
+
+        assert descriptor1 =~ ~r/Extended options: /
+        assert descriptor2 =~ ~r/minimum/
+        assert descriptor2 =~ ~r/maximum/
+        assert descriptor2 =~ ~r/sample size/
+        assert descriptor2 =~ ~r/mode/
```

Comment:
I somehow like the terminology `header` better for `descriptor2` - wdyt?

Reply:
Agreed, modified accordingly